features/upgrade/api_auth/upgrade.feature

```diff
@@ -0,0 +1,56 @@
+Feature: apiserver and auth related upgrade check
+  # @author kewang@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Default RBAC role, rolebinding, clusterrole and clusterrolebinding without any missing after upgraded - prepare
+    When I run the :get admin command with:
+      | resource | clusterroles.rbac |
+      | o        | yaml              |
+    Then the output is parsed as YAML
+    And the output should contain:
+      | autoupdate: "true" |
+    And the output should not contain:
+      | autoupdate: "false" |
+    # Make some changes on clusterrole resources before upgrade
+    Given as admin I successfully patch resource "clusterrole.rbac/system:build-strategy-docker" with:
+      | {"rules": [{"apiGroups": ["","build.openshift.io"],"resources": ["builds/docker","builds/optimizeddocker"],"verbs": [ "get" ]}] } |
+    When I run the :get admin command with:
+      | resource      | clusterroles.rbac            |
+      | resource_name | system:build-strategy-docker |
+      | o             | yaml                         |
+    Then the output is parsed as YAML
+    And the expression should be true> @result[:parsed]['rules'][0]['verbs'][0] != "create"
+    And the expression should be true> @result[:parsed]['rules'][0]['verbs'][0] == "get"
```

Comment:
Strictly speaking, [0] **==** "get" definitely means it will **!=** other string

Reply:
@wangke19 @rhpmali , tips for you about quickly testing ruby when you want to try like string operation:
$ irb
irb(main):013:0> verbs=['get']
=> ["get"]
irb(main):014:0> verbs.include? 'get'
=> true
irb(main):015:0> verbs.include? 'create'
=> false
This can used for string issue like https://github.com/openshift/verification-tests/pull/1054#discussion_r422443145 too.

Ke, please update  `And the expression should be true> @result[:parsed]['rules'][0]['verbs'][0] != "create"` to:
And the expression should be **false**> @result[:parsed]['rules'][0]['verbs']include? "create"

Reply:
Updated to include method to handle.